index.html

```diff
@@ -7,9 +7,56 @@
 <body>
 	<main>
 		<section class="cta">
-			<h1></h1>
-			<p></p>
+			<h1>SIGN UP FOR A FREE CONSULTATION</h1>
+			<p>Remove the stress of negotiating getting the best car price</p>
+		<div id="contact_form">
+    <form name="form1" id="ff" method="post" action="contact.php">
+        <label>
+        <span>Name:</span>
+        <input type="text" placeholder="Name" name="name" id="name" required autofocus>
+        </label>
+         
+        <label>
+        <span>Phone:</span>
+        <input type="tel" placeholder="Phone Number" name="phone" id="phone" required>
+        </label>
+         
+        <label>
+        <span>Email:</span>
+        <input type="email" placeholder="Email" name="email" id="email" required>
+          </label>
+       
+        <input class="sendButton" type="submit" name="Submit" value="Send">
+             
+    </form>
+    </div>
+    <p>OR CALL</p>
+    <p>860.555.1212</p>
```

Comment:
This would be better to use an "a" tag <a href="tel:555-555-5555">555-555-5555</a>